mobile-bridge-api.json

```diff
@@ -389,6 +389,18 @@
             "documentation": "The additional context to send."
           }
         ]
+      },
+      {
```

Comment:
In `dd-ios-sdk` RUM session sampling is [configured for the `Datadog` facade](https://github.com/DataDog/dd-sdk-ios/blob/db69af6eda2abef842685a24893e7e1e42bd1f0d/api-surface-swift#L82), not for the `RUMMonitor` instance. I [find it similar for `dd-android-sdk`](https://github.com/DataDog/dd-sdk-android/blob/776853a981e84b48384e52cb266dfffaa33b0f80/dd-sdk-android/apiSurface#L98), although Android SDK also [provides this setting for `RUMMonitor`](https://github.com/DataDog/dd-sdk-android/blob/776853a981e84b48384e52cb266dfffaa33b0f80/dd-sdk-android/apiSurface#L308).

The common denominator for both native SDKs is to configure it in SDK configuration, not per `RUMMonitor`. This would be easier if we expose this API in `Dd` facade - otherwise we'd need to add it for the `RUMMonitor` on iOS.

Reply:
Do you mean making this a part of the `DdSdk` object?

Reply:
Yes, in the same way as it is part of `Datadog.initialize(configuration:)` now. WDYT?

Reply:
For now let's make it part of the initialization configuration, it's going to be simpler for everyone